components/script/dom/webidls/ExtendableMessageEvent.webidl

```diff
@@ -20,5 +20,5 @@ dictionary ExtendableMessageEventInit : ExtendableEventInit {
   DOMString origin;
   DOMString lastEventId;
   // (Client or ServiceWorker /*or MessagePort*/)? source;
-  // sequence<MessagePort>? ports;
+  sequence<MessagePort> ports;
```

Comment:
We can add a bunch of default values to the init, so that it matches https://w3c.github.io/ServiceWorker/#extendablemessageevent

For example:
- `sequence<MessagePort> ports = [];`
- `DOMString origin = "";`

You can look at `MessageEvent.webidl` which does the same thing. 

That way you can remove the various `unwrap` following the call to `init` above.